src/content/packages-and-plugins/developing-packages.md

```diff
@@ -51,7 +51,7 @@ Packages can contain more than one kind of content:
   see the Medium article by Harry Terkelsen,
   [How to Write a Flutter Web Plugin, Part 1][].
 
-**FFI Plugin packages**
+**FFI packages**
```

Comment:
I don't think that deleting plugin enhances clarity. A plugin is a special type of package. This is a global comment, btw. :)